Google.PowerShell/Compute/GceHealthCheckCmdlets.cs

```diff
@@ -0,0 +1,545 @@
+﻿using Google.Apis.Compute.v1;
+using Google.Apis.Compute.v1.Data;
+using Google.PowerShell.Common;
+using System;
+using System.Collections.Generic;
+using System.Management.Automation;
+
+namespace Google.PowerShell.ComputeEngine
+{
+    /// <summary>
+    /// <para type="synopsis">
+    /// Adds a Google Compute Engine health check.
+    /// </para>
+    /// <para type="descritpion">
+    /// Adds a Google Compute Engine health check. Use this cmdlet to create both http and https health checks.
+    /// </para>
+    /// </summary>
+    [Cmdlet(VerbsCommon.Add, "GceHealthCheck", DefaultParameterSetName = ParameterSetNames.ByValues)]
+    [OutputType(typeof(HttpHealthCheck), typeof(HttpsHealthCheck))]
+    public class AddGceHealthCheckCmdlet : GceConcurrentCmdlet
+    {
+        private class ParameterSetNames
+        {
+            public const string ByValues = "ByValues";
+            public const string ByHttpObject = "ByHttpObject";
+            public const string ByHttpsObject = "ByHttpsObject";
+        }
+
+        /// <summary>
+        /// <para type="description">
+        /// The project to add the health check to. Defaults to the project in the Cloud SDK config.
+        /// </para>
+        /// </summary>
+        [Parameter]
+        [ConfigPropertyName(CloudSdkSettings.CommonProperties.Project)]
+        public string Project { get; set; }
+
+        /// <summary>
+        /// <para type="description">
+        /// The name of the health check to add.
+        /// </para>
+        /// </summary>
+        [Parameter(ParameterSetName = ParameterSetNames.ByValues, Mandatory = true, Position = 0)]
+        public string Name { get; set; }
+
+        /// <summary>
+        /// <para type="description">
+        /// Human readable description of the health check.
+        /// </para>
+        /// </summary>
+        [Parameter(ParameterSetName = ParameterSetNames.ByValues)]
+        public string Description { get; set; }
+
+        /// <summary>
+        /// <para type="description">
+        /// The value of the host header in the health check request. If left empty
```

Comment:
"If left empty (default value)" isn't cromulent. Could you rephrase this to be easier to parse?